resources/qml/Cura.qml

```diff
@@ -826,6 +826,8 @@ UM.MainWindow
     {
         id: addMachineDialog
         title: catalog.i18nc("@title:window", "Add Printer")
+        maximumWidth: undefined
+        maximumHeight: undefined
```

Comment:
these two values gives QML warnings as it expects integer value. 
I don't know if its too much to ask. but is it possible to create it like a resizable window?

Reply:
It took me a while to find the proper way to do it, without changing the default behavior of WizardDialog. Hope you'll like it :)